CHANGELOG.md

```diff
@@ -3,6 +3,28 @@
 All notable changes to this project will be documented in this file. The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org).
 
+## Supported Release 4.19.0
+### Summary
+
+This release adds new functions and better documentation/fixes for existing functions. With a noteworthy fixes to file_line.
```

Comment:
Maybe something more like "This release adds new functions and better documentation/fixes for existing functions with a noteworthy fix to file_line."